src/draw/lv_draw_private.h

```diff
@@ -176,7 +176,7 @@ struct _lv_draw_unit_t {
 typedef struct {
     lv_draw_unit_t * unit_head;
     uint32_t unit_cnt;
-    uint32_t used_memory_for_layers_kb;
+    uint32_t used_memory_for_layers;
```

Comment:
Please add comment to explain the unit is kB

Reply:
The units are now bytes. I added a comment.